### PR TITLE
SSH Ansible Content

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

**Comment by mpreisler** [comment]
Wow, awesome work! Kudos.

I will review this sometime today unless somebody else takes it.

**Comment by mpreisler** [comment]
This PR has a conflict with https://github.com/OpenSCAP/scap-security-guide/pull/2010/files but the latter uses a variable and a different ssh config key. @shawndwells could you take a look?

**Comment by mpreisler** [comment]
The ansible proposed by Shawn is consistent with our bash remediation. In case the other PR is correct we should fix both bash and ansible remediations for sshd keep alive.

**Comment by mpreisler** [comment]
ACK

File: shared/templates/static/ansible/sshd_disable_empty_passwords.yml [hunk 1]
```diff
@@ -3,12 +3,11 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
-- name: "Disable SSH Access via Empty Passwords"
+- name: "Diable SSH Access via Empty Passwords"
```
Comment (konstruktoid) [inline shared/templates/static/ansible/sshd_disable_empty_passwords.yml]: Typo.

**Pull request merged**